app/assets/stylesheets/application.css.scss

```diff
@@ -533,4 +533,59 @@ span.twitter-typeahead {
 
 .latest-run--ok {
   border-color: #5cb85c;
+}
+
+.alert-wrap {
+  text-align: center;
+  margin-bottom: 1.5em;
+
+  .alert {
+    display: inline-block;
+    margin: 0 auto;
+  }
+}
+
+.alert-icon {
+  &.alert-success,
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets